app/controllers/breweries_controller.rb

```diff
@@ -0,0 +1,11 @@
+class BreweriesController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    @brewery = Brewery.find(params[:id])
+  end
+
```

Comment:
This should be a private method. Also, if you're not creating / editing, then why do you need strong params here?